Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -187,24 +187,25 @@ function Get-SqlPackageCommandArguments
 
 function Run-Command
 {
-    param([String][Parameter(Mandatory=$true)] $command)
-
-    try
-	{
-        if( $psversiontable.PSVersion.Major -le 4)
-        {
-           cmd.exe /c "`"$command`"" 2>&1
-        }
-        else
-        {
-           cmd.exe /c "$command" 2>&1
-        }
-
+    param(
+        [string]$command,
+        [bool] $failOnErr = $true
+    )
+    $ErrorActionPreference = 'Continue'
+    if( $psversiontable.PSVersion.Major -le 4)
+    {
+        $result = cmd.exe /c "`"$command`"" 2>&1
     }
-	catch [System.Exception]
+    else
+    {
```

Comment:
If some Exception is raised here , We wont be returning result or exception trace in that case, right ?